src/__experimental__/components/input-icon-toggle/input-icon-toggle-classic.style.js

```diff
@@ -0,0 +1,26 @@
+import { css } from 'styled-components';
+import { THEMES } from '../../../style/themes';
+
+export default ({
+  inputFocus, inputHover, theme, type
+}) => theme.name === THEMES.classic && css`
+  background-color: #e6ebed;
+  border-left: 1px solid #bfccd2;
+  box-sizing: border-box;
+  color: #003349;
+  cursor: pointer;
```

Comment:
this property is already defined in the `input-icon-toggle.style`

Reply:
Removed 👍